__tests__/utils.test.js

```diff
@@ -0,0 +1,17 @@
+const { stringToDeviceIDType, DeviceIdType } = require("../Utils.js");
```

Comment:
add more comments how purpose of the test etc

Reply:
done